openapi/components/schemas/StorefrontQuoteChangeOrder.yaml

```
@@ -20,44 +16,38 @@ properties:
   action:
     description: |-
       Action of the quote for change order items.
-      On accept quote interim invoice is issued and linked to `invoiceId`.
-      After the invoice is paid changed items are applied to an order with `orderId`.
+      On acceptance quote items are applied to the related order.
```

suggestion: for readability
```suggestion
      When a quote is accepted, quote items are applied to the related order.
```

---

Thanks, will change everywhere.